src/sage/numerical/optimize.py

```diff
@@ -400,7 +400,7 @@ def minimize(func, x0, gradient=None, hessian=None, algorithm='default',
     from sage.ext.fast_callable import fast_callable
     import numpy
     if int(numpy.version.short_version[0]) > 1:
-        numpy.set_printoptions(legacy="1.25")
+        _ = numpy.set_printoptions(legacy="1.25")
```

Comment:
What's the point of these changes in normal python code? The return value should be ignored either way, or not?

Reply:
Ugh, must be a regex going wrong. Will fix soon